survey/survey-domain/src/main/java/me/nalab/survey/domain/feedback/FormQuestionFeedbackable.java

```diff
@@ -25,9 +25,14 @@ public void setRead(boolean read) {
 		isRead = read;
 	}
 
+	public void switchBookmark() {
+		this.getBookmark().replaceIsBookmarked();
+		this.getBookmark().updateBookmarkedAt();
+	}
```

Comment:
```suggestion
	public void switchBookmark() {
		bookmark.replaceIsBookmarked();
		bookmark.updateBookmarkedAt();
	}
```
이런 느낌?! 생각했고
updateBookmarkedAt()이 replaceBookmarked 안으로 들어가야지 외부에서 두번 호출할 필요 없이 북마크 상태 변경하고 상태 변경 시점 기록이라는 2가지 일이 하나의 원자성으로 보장되는 듯?!

Reply:
아 아예 두 작업을 하나의 원자성 단위로 생각했구나

Reply:
응응 나도 그게 더 맞는것같애!!